tests/tensor_search/test_multimodal_tensor_combination.py

```diff
@@ -229,12 +238,144 @@ def test_multimodal_tensor_combination_tensor_value(self):
             np.array(tensor_search.get_document_by_id(config=self.config, index_name=self.index_name_1, document_id="4",
                                                       show_vectors=True)['_tensor_facets'][0]["_embedding"])
 
-        expected_tensor = np.mean([text_tensor_1 * 0.32, text_tensor_2 * 0, image_tensor_1 * -0.48, image_tensor_2 * 1.34], axis = 0)
+        expected_tensor = np.mean(
+            [text_tensor_1 * 0.32, text_tensor_2 * 0, image_tensor_1 * -0.48, image_tensor_2 * 1.34], axis=0)
         assert np.allclose(combo_tensor_1, expected_tensor, atol=1e-5)
         assert np.allclose(combo_tensor_2, expected_tensor, atol=1e-5)
         assert np.allclose(combo_tensor_3, expected_tensor, atol=1e-5)
         assert np.allclose(combo_tensor_4, expected_tensor, atol=1e-5)
 
+    def test_multimodal_tensor_combination_tensor_value_different_mappings(self):
```

Comment:
This is a good test. Can we do another one where each mapping is a completely different (no shared subfields)?

Reply:
added